CMakeLists.txt

```diff
@@ -540,6 +540,13 @@ if(ANDROID)
   include_directories(Source/Android)
 endif()
 
+if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
- Is the intention to expose this for all externals?
- Does this need to be `BEFORE`, since it tries to delegate to the system headers anyways?